rclcpp_lifecycle/include/rclcpp_lifecycle/lifecycle_node.hpp

```diff
@@ -493,170 +537,324 @@ class LifecycleNode : public node_interfaces::LifecycleNodeInterface,
   get_node_clock_interface();
 
   /// Return the Node's internal NodeGraphInterface implementation.
+  /**
+   * \sa rclcpp::Node::get_node_graph_interface
+   */
   RCLCPP_LIFECYCLE_PUBLIC
   rclcpp::node_interfaces::NodeGraphInterface::SharedPtr
   get_node_graph_interface();
 
   /// Return the Node's internal NodeLoggingInterface implementation.
+  /**
+   * \sa rclcpp::Node::get_node_logging_interface
+   */
   RCLCPP_LIFECYCLE_PUBLIC
   rclcpp::node_interfaces::NodeLoggingInterface::SharedPtr
   get_node_logging_interface();
 
   /// Return the Node's internal NodeTimersInterface implementation.
+  /**
+   * \sa rclcpp::Node::get_node_timers_interface
+   */
   RCLCPP_LIFECYCLE_PUBLIC
   rclcpp::node_interfaces::NodeTimersInterface::SharedPtr
   get_node_timers_interface();
 
   /// Return the Node's internal NodeTopicsInterface implementation.
+  /**
+   * \sa rclcpp::Node::get_node_topics_interface
+   */
   RCLCPP_LIFECYCLE_PUBLIC
   rclcpp::node_interfaces::NodeTopicsInterface::SharedPtr
   get_node_topics_interface();
 
   /// Return the Node's internal NodeServicesInterface implementation.
+  /**
+   * \sa rclcpp::Node::get_node_services_interface
+   */
   RCLCPP_LIFECYCLE_PUBLIC
   rclcpp::node_interfaces::NodeServicesInterface::SharedPtr
   get_node_services_interface();
 
   /// Return the Node's internal NodeParametersInterface implementation.
+  /**
+   * \sa rclcpp::Node::get_node_parameters_interface
+   */
   RCLCPP_LIFECYCLE_PUBLIC
   rclcpp::node_interfaces::NodeParametersInterface::SharedPtr
   get_node_parameters_interface();
 
   /// Return the Node's internal NodeParametersInterface implementation.
+  /**
+   * \sa rclcpp::Node::get_node_time_source_interface
+   */
   RCLCPP_LIFECYCLE_PUBLIC
   rclcpp::node_interfaces::NodeTimeSourceInterface::SharedPtr
   get_node_time_source_interface();
 
   /// Return the Node's internal NodeWaitablesInterface implementation.
+  /**
+   * \sa rclcpp::Node::get_node_waitables_interface
+   */
   RCLCPP_LIFECYCLE_PUBLIC
   rclcpp::node_interfaces::NodeWaitablesInterface::SharedPtr
   get_node_waitables_interface();
 
   /// Return the NodeOptions used when creating this node.
+  /**
+   * \sa rclcpp::Node::get_node_options
+   */
   RCLCPP_LIFECYCLE_PUBLIC
   const rclcpp::NodeOptions &
   get_node_options() const;
 
   //
   // LIFECYCLE COMPONENTS
   //
+  /// Return the current State.
+  /**
+   * \return the current state
+   */
   RCLCPP_LIFECYCLE_PUBLIC
   const State &
   get_current_state();
 
+  /// Return a list with the available states.
+  /**
+   * \return list with the available states.
+   */
   RCLCPP_LIFECYCLE_PUBLIC
   std::vector<State>
   get_available_states();
 
+  /// Return a list with the available transitions.
+  /**
+   * \return list with the available transitions.
+   */
   RCLCPP_LIFECYCLE_PUBLIC
   std::vector<Transition>
   get_available_transitions();
 
-  /// trigger the specified transition
+  /// trigger the specified transition.
```

Comment:
```suggestion
  /// Trigger the specified transition.
```

Capitalize this one and all the other briefs below.